config/samples/csi-baremetal_v1_deployment.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     app.kubernetes.io/managed-by: csi-baremetal-operator
     app.kubernetes.io/version: 1.0.0
 spec:
+  platform: "openshift"
```

Comment:
Please create new sample for `openshift`.

Reply:
added